### PR TITLE
update registration

### PR DESCRIPTION
**Pull request by mritunjaysaha**
Hey Maintainer,

I've completed 5 projects and 3 blogs

### CODE REVIEW

File: content/participants/ttjc@mritunjaysaha/index.md [hunk 1]
```diff
@@ -28,18 +27,81 @@ I don't give up easily.
...
+
+_github link_ https://github.com/mritunjaysaha/device-tracker
+
+#### Bluetooth Device Tracker
+
+_description_ Keeps track of the selected android devices within the bluetooth range
+
+_stack_ Java
+
+_github link_ https://github.com/mritunjaysaha/android-bt-v1
```
Comment (gkanishk) [inline content/participants/ttjc@mritunjaysaha/index.md:77]: Where is the hosted link!!
Please publish the app on playstore and provide the link.

**Comment by gkanishk** [comment]
Hey @mritunjaysaha ,
Why have you removed that android project?
Aren't you publishing it? and what about your fifth project then?

**Comment by mritunjaysaha** [comment]
Hey @gkanishk ,
I made that android app as part of my Diploma project. It was quite basic but took a lot of my time. I need to make some more changes to the UI  and create a logo before publishing it. I have removed it, so I don't have a fifth project now.

**Comment by gkanishk** [comment]
So are you making one? Make any basic project as the fifth one take your time you can do it by today or tomorrow please add it as you have almost completed the challenge.

**Comment by mritunjaysaha** [comment]
Ok. I'll add another project by tomorrow. Thanks for the extra day.

**Comment by mritunjaysaha** [comment]
Hey @gkanishk ,
I've completed my 5th project.

**Comment by gkanishk** [comment]
Awesome! You can raise  a **PR** for the finalist refer [faq](https://2020.teamtanay.jobchallenge.dev/finalist-entry/) or #935

**Pull request merged**